ocean_lib/models/data_nft.py

```diff
@@ -119,6 +124,8 @@ def deploy_contract(self, config_dict, tx_dict) -> DataNFT:
             self.owner or wallet_address,
             tx_dict,
         )
+        if "required_confs" in tx_dict.keys():
```

Comment:
please remove this. This is a test-related dependency an shouldn't have a different return type. It's bad coupling.